style.css

```diff
@@ -0,0 +1,63 @@
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+}
+
+#hamburger {
+  margin-top: 15px;
+  margin-right: 16px;
+}
+
+body {
+  margin: 0%;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  height: 600px;
+  background-image: url(/images/image_geometry.svg);
```

Comment:
Same here the background is not rendering. Kindly edit the src attribute adding the `.` at the begging of the path. :eyes: :eyes: 
![image](https://user-images.githubusercontent.com/98363075/198165539-d7afc51b-551e-4a7a-8d06-fd4cd428923a.png)